specs/eip4844/polynomial-commitments.md

```diff
@@ -336,23 +350,29 @@ def blob_to_kzg_commitment(blob: Blob) -> KZGCommitment:
 #### `verify_kzg_proof`
 
 ```python
-def verify_kzg_proof(polynomial_kzg: KZGCommitment,
+def verify_kzg_proof(commitment: Bytes48,
                      z: Bytes32,
                      y: Bytes32,
-                     kzg_proof: KZGProof) -> bool:
+                     kzg_proof: Bytes48) -> bool:
```

Comment:
Nit: I think we should either rename `commitment` to `kzg_commitment` or rename `kzg_proof` to `proof`. My vote goes to renaming `kzg_proof` to `proof` throughout the file.

Reply:
Addressed in af99c1924